a-short-introduction-to-kubernetes/index.md

```diff
@@ -0,0 +1,372 @@
+---
+title: 'Introduction to Kubernetes'
+description: 'A short introduction to Kubernetes in a Directus context'
+author:
+  name: 'Mikke Schirén'
+  avatar_file_name: 'mikke.png'
+---
+
+## Introduction
+
+Originated in Googles project Borg, which started in 2003, Kubernetes (K8s) is today the most popular open-source container orchestration system, and it automates Deployments, scaling and management of applications. And Kubernetes is in many ways, a complicated beast to handle, like a puzzle with many pieces. If you never used Kubernetes before, it is maybe not the easiest way forward for your self-hosted Directus project. But there a lot of gains with Kubernetes running an application like Directus.
+
+Good to remember that Kubernetes is not a traditional, all-inclusive PaaS (Platform as a Service) system, when you start up a new cluster, depending on which flavor your using, you need to install some, or all pieces yourself.
```

Comment:
I'd suggest removing this - we haven't spoken about it in the context of a PaaS so I don't think we need to compare. And by introducing the problem it solves upfront, we make it easier to understand immediately.